extra/DEVELOPER_DOCS/Listeners.md

```diff
@@ -216,5 +220,12 @@ new_model = tok2vec_model.attrs["replace_listener"](new_model)
 ```
 
 The new config and model are then properly stored on the `nlp` object.
-Note that this functionality (running the replacement for a transformer listener) was broken prior to 
+Note that this functionality (running the replacement for a transformer listener) was broken prior to
 `spacy-transformers` 1.0.5.
+
+As of spaCy 3.7, the `replace_listener` callback accepts three arguments instead of just one: the new (copied) model,
```

Comment:
The text here is a bit ambiguous, since the callback can also accept just one argument. Maybe it's best to write out both cases in the docs here? The listener mechanism is pretty tricky, so I think it's good to be precise in these docs.

Reply:
Clarified that sentence.